packages/common/src/store/ui/related-artists/sagas.ts

```diff
@@ -19,19 +20,23 @@ const getUserId = accountSelectors.getUserId
 
 export function* fetchRelatedArtists(action: PayloadAction<{ artistId: ID }>) {
   yield* waitForRead()
-  const apiClient = yield* getContext('apiClient')
+  const sdk = yield* getSDK()
   const remoteConfigInstance = yield* getContext('remoteConfigInstance')
+  const currentUserId = yield* select(getUserId)
   if (relatedArtistsActions.fetchRelatedArtists.match(action)) {
     const artistId = action.payload.artistId
 
-    const relatedArtists = yield* call(
-      [apiClient, apiClient.getRelatedArtists],
+    const { data } = yield* call(
+      [sdk.full.users, sdk.full.users.getRelatedUsers],
```

Comment:
I wonder if we even need the context argument anymore or if you could just do 

```suggestion
    const { data } = yield* call(
      sdk.full.users.getRelatedUser,
```

Reply:
Unfortunately I think we do because each SDK api is an instance of a class and makes copious use of `this`.